File: scienceCenterLukaFrontend/src/app/model/reviewing-editor-dto.ts
import { Article } from './article';

import { Magazine } from './magazine';

import { OpinionAboutArticle } from './opinion-about-article';

export class ReviewingEditorDto{

    article: Article;
    magazine: Magazine;
    opinions: OpinionAboutArticle[];
    authorsMessages: OpinionAboutArticle[];

    editorOpinion: OpinionAboutArticle;

    fields: any[];
    fieldResults: any[];

   constructor(){}
}

File: scienceCenterLuka/src/main/java/com/project/dto/ReviewingEditorDto.java
package com.project.dto;

import java.io.Serializable;
import java.util.List;

import org.camunda.bpm.engine.form.FormField;

import com.project.model.OpinionAboutArticle;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReviewingEditorDto implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = -8111994278994400192L;
	
	private ArticleDto article;
	private MagazineDto magazine;
	private List<OpinionAboutArticle> opinions;
	
	private List<OpinionAboutArticle> authorsMessages;

	
	private OpinionAboutArticle editorOpinion;
	
	private List<FormField> fields;
	private List<FormSubmissionDto> fieldResults;

}
